content/blog/2020-08-10/august-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,205 @@
+---
+title: August '20 Heartbeat
+date: 2020-08-10
+description: |
+  Catch our monthly updates- featuring the CML release, DVC meetup recap, 
+  a new video tutorial series, and the best reading about pipelines and
+  DataOps.
+descriptionLong: |
+  Catch our monthly updates- featuring the CML release, DVC meetup recap, 
+  a new video tutorial series, and the best reading about pipelines and
+  DataOps.
+picture: 2020-08-10/header.png
+pictureComment: DeeVee avoids the summer sun at Mount Rainier National Park.
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/august-20-heartbeat/465
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - Meetup
+---
+
+Welcome to our August roundup of cool news, new releases, and recommended
+reading in the MLOps world!
+
+## News
+
+### CML release
+
+At the beginning of July, we went live with a new project: Continuous Machine
+Learning, or CML for short. If you hadven't heard, CML is an open-source toolkit
```

Comment:
link to CML - site and/or github would be helpful

Reply:
added